diff --git a/src/map/itemdb.hpp b/src/map/itemdb.hpp
index 6cf29f507..c65ede55a 100644
--- a/src/map/itemdb.hpp
+++ b/src/map/itemdb.hpp
@@ -2972,6 +2972,8 @@ enum e_random_item_group {
 	IG_REGIA_HUNTING_SET,
 	IG_MYSTERIOUS_FRUIT_BOX,
 	IG_ZONDA_SUPPORT_PACKAGE,
+	IG_YOR_CARD,
+	IG_TAN_BOX,
 
 	IG_MAX,
 };
diff --git a/src/map/script_constants.hpp b/src/map/script_constants.hpp
index 08e4d9625..8337a6190 100644
--- a/src/map/script_constants.hpp
+++ b/src/map/script_constants.hpp
@@ -7072,6 +7072,8 @@
 	export_constant(IG_REGIA_HUNTING_SET);
 	export_constant(IG_MYSTERIOUS_FRUIT_BOX);
 	export_constant(IG_ZONDA_SUPPORT_PACKAGE);
+	export_constant(IG_YOR_CARD);
+	export_constant(IG_TAN_BOX);
 
 	/* unit stop walking */
 	export_constant(USW_NONE);
